packages/block-library/src/comments-pagination-previous/block.json

```diff
@@ -38,6 +38,23 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
```suggestion
```

Applying borders to this block is more of a niche use case. It might be best to hide these controls by default. 

Unlike for the spacing support, here we can simply omit the `__experimentalDefaultControls` values.